src/components/dialogs/PostInteractionSettingsDialog.tsx

```diff
@@ -385,104 +390,228 @@ export function PostInteractionSettingsForm({
+          <View style={[a.gap_sm, {opacity: replySettingsDisabled ? 0.3 : 1}]}>
+            <Text style={[a.text_md, a.font_medium]}>
+              <Trans>Who can reply</Trans>
             </Text>
 
-            <Text style={[a.pt_sm, t.atoms.text_contrast_medium]}>
-              <Trans>Allow replies from:</Trans>
-            </Text>
-
-            <View style={[a.flex_row, a.gap_sm]}>
-              <Selectable
-                label={_(msg`Everybody`)}
-                isSelected={
-                  !!threadgateAllowUISettings.find(v => v.type === 'everybody')
-                }
-                onPress={() =>
+            <Toggle.Group
+              label={_(msg`Set who can reply to your post`)}
+              type="radio"
+              maxSelections={1}
+              disabled={replySettingsDisabled}
+              values={
+                everyoneCanReply
+                  ? ['everyone']
+                  : noOneCanReply
+                    ? ['nobody']
+                    : []
+              }
+              onChange={val => {
+                if (val.includes('everyone')) {
                   onChangeThreadgateAllowUISettings([{type: 'everybody'}])
-                }
-                style={{flex: 1}}
-                disabled={replySettingsDisabled}
-              />
-              <Selectable
-                label={_(msg`Nobody`)}
-                isSelected={noOneCanReply}
-                onPress={() =>
+                } else if (val.includes('nobody')) {
                   onChangeThreadgateAllowUISettings([{type: 'nobody'}])
+                } else {
+                  onChangeThreadgateAllowUISettings([{type: 'mention'}])
                 }
-                style={{flex: 1}}
-                disabled={replySettingsDisabled}
-              />
-            </View>
-
-            {!noOneCanReply && (
-              <>
-                <Text style={[a.pt_sm, t.atoms.text_contrast_medium]}>
-                  <Trans>Or combine these options:</Trans>
-                </Text>
-
-                <View style={[a.gap_sm]}>
-                  <Selectable
-                    label={_(msg`Mentioned users`)}
-                    isSelected={
-                      !!threadgateAllowUISettings.find(
-                        v => v.type === 'mention',
-                      )
-                    }
-                    onPress={() => onPressAudience({type: 'mention'})}
-                    disabled={replySettingsDisabled}
-                  />
-                  <Selectable
-                    label={_(msg`Users you follow`)}
-                    isSelected={
-                      !!threadgateAllowUISettings.find(
-                        v => v.type === 'following',
-                      )
+              }}>
+              <View style={[a.flex_row, a.gap_sm]}>
+                <Toggle.Item
+                  name="everyone"
+                  type="checkbox"
+                  label={_(msg`Allow anybody to reply`)}
```

Comment:
```suggestion
                  label={_(msg`Allow anyone to reply`)}
```
nit: Suggest `anyone` to align with main string